src/lib/db-listeners/db-docs-content-listener.ts

```diff
@@ -69,6 +69,7 @@ export class DBDocumentsContentListener extends BaseListener {
           return;
         }
 
+        // RESEARCHER-ACCESS: should this be changed to isTeacherOrResearcher?
```

Comment:
This would be 'yes' if we want the researcher view to live-update with all student work.